docs/pages/blog/turbo-1-10-0.mdx

```diff
@@ -0,0 +1,137 @@
+    },
+    "test": {
+      "dotEnv": [".env.test.local", ".env.test", ".env"]
+    }
+  }
+}
+```
+
+<Callout>
+  This feature does **not** handle system environment variables. If you are not using `.env` files outside of development, make sure your environment variables are still enumerated in either [env](/repo/docs/core-concepts/caching/environment-variable-inputs#pipelinetaskenv), or [globalEnv](/repo/docs/core-concepts/caching/environment-variable-inputs#globalenv) to ensure they are included in your hash.
+</Callout>
+
+For more, check out the [documentation](/repo/docs/core-concepts/caching/environment-variable-inputs#env-files).
+
+### Environment Variable Wildcards
+
+Large applications can end up with many environment variables in their `turbo.json`. This can make maintaining your `turbo.json` extremely difficult to manage and maintain.
+
+With wildcards, you can now specify patterns of variables to include in your hash.
+
+```json
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "env": ["ACME_*"]
+    }
+  }
+}
+```
+
+For more, check out the [documentation](/repo/docs/core-concepts/caching/environment-variable-inputs#wildcards).
+
+### Environment modes
+
+Turborepo environment modes are now out of [experimental mode](/repo/docs/faq#what-does-experimental-mean) and are available for general use.
+
+By default, and when turbo is invoked with --env-mode=loose , all environment variables from the machine are made available to every single turbo task.
+This ensures the greatest compatibility while accepting some risk that a task will implicitly have access to a environment variable that is not specified in your `turbo.json`.
+
+In the new strict mode, when turbo is invoked with --env-mode=strict , only [important system environment variables](/repo/docs/core-concepts/caching/environment-variable-inputs#system-environment-variables) and environment variables enumerated inside of turbo.json will be made available to a task.
+
+For more, check out the [documentation](/repo/docs/core-concepts/caching/environment-variable-inputs#loose--strict-environment-modes).
+
+## Community
+
+Since releasing [Turborepo v1.9](/blog/turbo-1-9-0) we've seen incredible adoption and community growth:
+
+- [21.5k+ GitHub Stars](https://github.com/vercel/turbo)
+- [1.2M+ weekly NPM downloads](https://www.npmjs.com/package/turbo)
+- 85 years of compute time saved through [Remote Caching on Vercel](https://vercel.com/docs/concepts/monorepos/remote-caching)
+
+Turborepo is the result of the combined work of all of its contributors, including our core team:
+
+- [Jared](https://github.com/jaredpalmer), [Greg](https://github.com/gsoltis), [Nathan](https://github.com/nathanhammond), [Thomas](https://github.com/tknickman), [Mehul](https://github.com/mehulkar), [Chris](https://github.com/chris-olszewski), [Nicholas](https://github.com/NicholasLYang), and [Alexander](https://github.com/arlyon)
```

Comment:
to match authors (except chris who doesn't have twitter)

```suggestion
- [Jared](https://twitter.com/jaredpalmer), [Greg](https://twitter.com/gsoltis), [Nathan](https://twitter.com/nathanhammond), [Thomas](https://twitter.com/tknickman), [Mehul](https://twitter.com/mehulkar), [Chris](https://github.com/chris-olszewski), [Nicholas](https://twitter.com/nicholaslyang), and [Alexander](https://twitter.com/_arlyon)
```
(not a big deal if we're matching nextjs, but i personally prefer twitter ref over gh :))

Reply:
I think we should keep github to match Next, and to ensure everyone has a link. Gives us all a good place to add our own info on our github profiles too.